### DISCUSSION TITLE
SonarQube API Import Error

### DISCUSSION THREAD

**Discussion by guerreroag12** [DefectDojo/django-DefectDojo#9608] [Q&A] [answered]
I am getting an error when running the Findings -> Import Scan Results with SonarQube API. I have set the API up as intended in the instructions. The error I am getting is this: An exception error occurred during the report import:'NoneType' object is not iterable

Any ideas as how I could fix it?

![Screenshot 2024-02-21 at 1 57 05 PM](https://github.com/DefectDojo/django-DefectDojo/assets/59993294/d988a001-9545-4f06-a96f-51f814ae05dd)

**Comment by manuel-sommer** [comment, marked as answer]
Hi @guerreroag12,

it looks like you did not choose any of your SonarQube configurations within the Engagement when you import the findings. Did you select "Settings --> Add API Scan Configuration" within the Engagmenet? 

If this still does not help, I guess you should rather ask this question within the OWASP DefectDojo slack channel in order to get an appropriate answer as this is not a discussion, but rather a call for help.

I hope this helped you.

Best,

Manuel
